org.nongnu.gsequencer.json

```diff
@@ -0,0 +1,241 @@
+	"--filesystem=host"
+    ],
+    "cleanup": [
+	"/include",
+	"/lib/pkgconfig",
+	"/share/man",
+	"/share/doc",
+	"/share/gtk-doc",
+	"*.la"
+    ],
+    "modules": [
+	"gtk2.json",
+	{
+	    "name": "ladspa",
+	    "no-autogen": true,
+	    "subdir": "src",
+	    "make-install-args": [
+		"INSTALL_PLUGINS_DIR=/app/lib/ladspa",
+		"INSTALL_INCLUDE_DIR=/app/include",
+		"INSTALL_BINARY_DIR=/app/bin"
+	    ],
+	    "cleanup": [
+		"/bin"
+	    ],
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "http://www.ladspa.org/download/ladspa_sdk_1.15.tgz",
+		    "sha256": "4229959b09d20c88c8c86f4aa76427843011705df22d9c28b38359fd1829fded"
+		}
+	    ]
+	},
+	
+	{
+	    "name": "fftw3f",
+	    "config-opts": [
+		"--enable-threads",
+		"--enable-shared",
+		"--enable-float",
+		"--disable-static"
+	    ],
+	    "build-options": {
+		"arch": {
+		    "x86_64": {
+			"config-opts": [
+			    "--enable-sse2",
+			    "--enable-avx",
+			    "--enable-avx-128-fma"
+			]
+		    }
+		}
+	    },
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "http://www.fftw.org/fftw-3.3.7.tar.gz",
+		    "sha256": "3b609b7feba5230e8f6dd8d245ddbefac324c5a6ae4186947670d9ac2cd25573"
+		}
+	    ],
+	    "cleanup": [
+		"/bin",
+		"/include",
+		"/lib/*.la"
+	    ]
+	},
+	
+	{
+	    "name": "jack2",
+	    "buildsystem": "simple",
+	    "build-commands": [
+		"./waf configure --prefix=/app --htmldir=/app/share/doc/jack/ --freebob=no --classic",
+		"./waf build -j $FLATPAK_BUILDER_N_JOBS",
+		"./waf install"
+	    ],
+	    "cleanup": [
+		"/bin",
+		"/share",
+		"/lib/jack",
+		"/lib/libjackserver.so*"
+	    ],
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "https://github.com/jackaudio/jack2/releases/download/v1.9.12/jack2-1.9.12.tar.gz",
+		    "sha256": "deefe2f936dc32f59ad3cef7e37276c2035ef8a024ca92118f35c9a292272e33"
+		}
+	    ]
+	},
+	
+	{
+	    "name": "liblo",
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "http://download.sf.net/sourceforge/liblo/liblo-0.29.tar.gz",
+		    "sha256": "ace1b4e234091425c150261d1ca7070cece48ee3c228a5612d048116d864c06a"
+		}
+	    ]
+	},      
+	{
+	    "name": "dssi",
+	    "rm-configure": true,
+	    "config-opts": [
+		"--disable-static"
+	    ],
+	    "cleanup": [
+		"/bin"
+	    ],
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "http://download.sf.net/sourceforge/dssi/dssi-1.1.1.tar.gz",
+		    "sha256": "f2c82b073a947c8255284249097667f9b14e660bf86186f3fcd3b3b3e087814e"
+		},
+		{
+		    "type": "script",
+		    "commands": [
+			"autoreconf -fiv"
+		    ],
+		    "dest-filename": "autogen.sh"
+		}
+	    ]
+	},
+
+	{
+	    "name": "audiofile",
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "http://audiofile.68k.org/audiofile-0.3.6.tar.gz",
+		    "sha256": "cdc60df19ab08bfe55344395739bb08f50fc15c92da3962fac334d3bff116965"
+		},
+		{
+		    "type": "patch",
+		    "path": "patches/audiofile-gcc6.patch"
```

Comment:
this file is missing and is causing the failed build.

Reply:
just added the missing patches